lib/Sema/TypeCheckPropertyWrapper.cpp

```diff
@@ -244,6 +244,30 @@ findSuitableWrapperInit(ASTContext &ctx, NominalTypeDecl *nominal,
   return viableInitializers.empty() ? nullptr : viableInitializers.front();
 }
 
+/// Returns true if the enclosingInstance parameter is `Never`,
+/// implying that there should be NO enclosing instance.
+static bool enclosingInstanceTypeIsNever(ASTContext &ctx, SubscriptDecl *subscript) {
+  if (!subscript)
+    return false;
+
+  ParameterList *indices = subscript->getIndices();
+  assert(indices && indices->size() > 0);
+
+  ParamDecl *param = indices->get(0);
+  if (param->getArgumentName() != ctx.Id_enclosingInstance)
+    return false;
+
+  auto paramTy = param->getType();
+  auto neverTy = ctx.getNeverType();
+  paramTy->dump();
+  paramTy->getCanonicalType()->dump();
+  neverTy->dump();
+  neverTy->getCanonicalType()->dump();
+  param->dump();
+  param->getType()->dump();
```

Comment:
Oh wait I forgot about these. I think this needs to get removed before merging into 5.5